docs/release/trg-5/trg-5-05.md

```diff
@@ -2,32 +2,33 @@
 title: TRG 5.05 - Chart Values
 ---
 
-| Status | Created      | Post-History    |
-|--------|--------------|-----------------|
-| Active | 10-Nov-2022  | Initial release |
+| Status | Created     | Post-History           |
+| ------ | ----------- | ---------------------- |
+| Update | 29-Feb-2024 | new examples and rules |
+| Active | 10-Nov-2022 | Initial release        |
 
 ## Why
 
 When external entities need to setup a whole Eclipse Tractus-X setup, they might need to configure multiply helm charts. To reduce errors and make it efficient overall, this guide aligns basic helm values across all products.
 
 ## Description
 
-Best practices for Helm chart `values.yaml` file.
+Rules and best practices for Helm chart `values.yaml` file.
 
 ## Container images
 
+- Image tag **must not** be set by default, instead it should use the
+  `appVersion` property from the `Chart.yaml` file
```

Comment:
I didn't noticed this at may first review and I wonder why define this in a such restrictive manner?
I mean it's good to default to the Charts.AppVersion if no image tag is defined but why not allow to set image tags as long as they are pointing to a versioned tag? Especially in the case of charts with multiple images this just causes unnecessary inflexibility.